wasm2c/wasm-rt-impl.c

```diff
@@ -42,6 +42,42 @@
 #define alloca _alloca
 #endif
 
+#if !defined(_WIN32)
+
+#include <pthread.h>
+#define WASM_RT_MUTEX_TYPE pthread_mutex_t
+#define WASM_RT_CREATE_MUTEX(lock) \
+  do {                             \
```

Comment:
Why not call pthread_mutex_init and pthread_mutex_destroy as part of these macros?   Seems maybe important, especially if you don't use PTHREAD_MUTEX_INITIALIZER.

Reply:
yikes, my bad! Will fix